packages/stories-web/src/components/progress-indicator/Circular.tsx

```diff
@@ -1,18 +1,22 @@
 import React, { FC } from 'react'
+import { Size } from '../types';
 
 export interface Props {
   max: number
   value?: number
+  size?: Extract<Size, 's' | 'm' | 'l'>
 }
 
 const Circular: FC<Props> = (props: Props) => {
   const {
     max,
     value,
+    size,
     ...rest
   } = props
 
   const classes = ['in-circular-progress-indicator']
+  classes.push(`-size-${size}`)
```

Comment:
```suggestion
  if (typeof size !== 'undefined') {
    classes.push(`-size-${size}`)
  }
```

sizeのデフォルト値をスタイル側で指定する様になったので、こちらのストーリーではあえてsizeを指定しないストーリーになったかと思います！
そのため、size propが指定されていない時に `-size-` が追加されないようにするのはいかがでしょうか！ 🙏